spec/lib/local-links-manager/check_links/link_status_requester_spec.rb

```diff
@@ -8,20 +8,26 @@
   subject(:link_status_requester) { described_class.new }
 
   context "links for enabled services" do
```

Comment:
Perhaps we should make it explicit in the context that we're checking homepage URLs as well?